### ISSUE TITLE
Unterminated string constant error due to \u2028 or \u2029 in string literal

### ISSUE THREAD

**Issue by chadunit** [fable-compiler/Fable#1283]
### Description

Including \u2028 or \u2029 in string literal results in error:

```
let str = "\u2028"
printfn "%s" str

ERROR in ./src/App.fs
Module parse failed: Unterminated string constant (337:17)
You may need an appropriate loader to handle this file type.
```
The escape sequence is presumably resolved to actual character in the compiled JS file, but the character is illegal: http://www.ecma-international.org/ecma-262/6.0/#sec-literals-string-literals

Workaround: Convert from int at runtime e.g. `char 0x2028`.

### Related information

* Fable version (`dotnet fable --version`): 1.3.4
* Operating system: Windows 10

**Comment by alfonsogarciacaro** [comment]
At one point Fable represented most of non-ASCII characters with unicode codes, but that was changed to fix #784. The illegal characters are only these ones? (taken from your link)

```
U+005C (REVERSE SOLIDUS), U+000D (CARRIAGE RETURN), U+2028 (LINE SEPARATOR), U+2029 (PARAGRAPH SEPARATOR), and U+000A (LINE FEED)
```

If that's the case, I guess we can track them in string literals and use the unicode notation.

**Comment by chadunit** [comment]
Yep I believe so, and the other 3 characters already work fine, so I think it's just these 2 to check for.
